packages/client/webapp/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docker-hub.uncharted.software/nginx:1.23-alpine
+
+# update packages and add some basics
+RUN apk update
+RUN apk add --no-cache curl && apk add --no-cache bash && apk add --no-cache vim
```

Comment:
Why do we need `&& apk add --no-cache vim`?
This is not for working on it, but for deployment right.

Reply:
this is for debugging deployments only.